wiki-fe/src/app/_components/NewArticle.jsx

```diff
@@ -0,0 +1,75 @@
+"use client"
+import React, { useState } from 'react';
+import { getSession } from '@/lib/session';
+import { useRouter } from 'next/navigation';
+import "@/styles/new-article.css";
+import ErrorMessage from "./auth/ErrorMessage";
+
+export default function NewArticle() {
+    const roleAdmin = 'admin';
+
+    const [text, setText] = useState('');
+    const [title, setTitle] = useState('');
+    const [errorMessage, setErrorMessage] = useState(null);
+
+    const router = useRouter();
+
+    const handleSubmit = async (event) => {
+        event.preventDefault();
+        try {
+            const session = await getSession();
+
+            if(session.role !== roleAdmin) {
+                return router.push('/');
+            }
+
+            await fetch('http://127.0.0.1:8000/article/create/', {
+                method: 'POST',
+                headers: { 
+                    'Content-Type': 'application/json',
+                },
+                body: JSON.stringify({
+                    title,
+                    text,
+                    user_id: session.userId
+                }), 
+            });
+        } catch (error) {
+            const data = await response.json();
+            setErrorMessage(data.error);
```

Comment:
essa mensagem de erro não vem do back, tem que colocar uma coisa genérica aqui
```suggestion
            setErrorMessage('Ocorreu um erro, tente novamente mais tarde.");
```